mapshader/utils.py

```diff
@@ -68,138 +68,138 @@ def psutil_fetching():
 
 def psutils_html():
     return '''
```

Comment:
Same thing here. I think that if We keep this in a real html file would be better for maintenance

Reply:
I've created a new issue for this #81.